<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScreenTestKit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidgetScreens"/>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="listWidgetTests"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStopTest"/>
   <addaction name="separator"/>
   <addaction name="actionLoop"/>
   <addaction name="actionAutoNext"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/play.svg</normaloff>:/resources/images/play.svg</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStartSelected">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/play.svg</normaloff>:/resources/images/play.svg</iconset>
   </property>
   <property name="text">
    <string>Start from Selected</string>
   </property>
  </action>
  <action name="actionStopTest">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/stop-button.svg</normaloff>:/resources/images/stop-button.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/checkbox-unchecked.svg</normaloff>
     <normalon>:/resources/images/checkbox-checked.svg</normalon>:/resources/images/checkbox-unchecked.svg</iconset>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
   <property name="toolTip">
    <string>Loop execution</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAutoNext">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/checkbox-unchecked.svg</normaloff>
     <normalon>:/resources/images/checkbox-checked.svg</normalon>:/resources/images/checkbox-unchecked.svg</iconset>
   </property>
   <property name="text">
    <string>Auto Next</string>
   </property>
   <property name="toolTip">
    <string>Auto Next Test</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
